src/ozmaps/mapmanager.c

```diff
@@ -60,16 +60,6 @@ int keysubtract(int, int);
 int gpstogpskey(uint64_t);
 int inthemap(int, PMTmapindex *);
 
-//  #define DEBUG
-#ifdef DEBUG
-
-#include "SDL2/SDL2_gfxPrimitives.h"
-#include <SDL2/SDL.h>
-#include <SDL2/SDL_image.h>
-#define SCREEN_WIDTH 500
-#define SCREEN_HEIGHT 500
-
-#endif
 
```

Comment:
note: If you follow the clang-format you should also remove one line, so there is only one line between the end of inthemap function and the comment.